opm/autodiff/WellInterface_impl.hpp

```diff
@@ -844,4 +844,11 @@ namespace Opm
         return 1.0;
     }
 
+    template<typename TypeTag>
+    void
+    WellInterface<TypeTag>::addWellContributions(Mat& mat) const
+    {
+        OPM_THROW(NotImplemented, "This well class does not support adding well contributions"
```

Comment:
Maybe this can be a pure virtual function?

Reply:
Then it would be necessary to implement this feature for multisegment wells immediately, and I think it's ok to do that in a second PR.

Reply:
Yes. You are right.

Reply:
This (unused) function is gone now.